src/pmi/src/pmi_wire.c

```diff
@@ -13,7 +13,7 @@
 
 #define IS_SPACE(c) ((c) == ' ')
 #define IS_EOL(c) ((c) == '\n' || (c) == '\0')
-#define IS_KEY(c) (isalnum(c) || (c) == '_' || (c) == '-')
+#define IS_KEY(c) (isalnum(c) || (c) == '_' || (c) == '-' || (c) == '/')
```

Comment:
are there any other differences in what is allowed that we know of?

Reply:
That's all AFAIK.